frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -108,9 +108,6 @@ function getParameterPanelBackgroundColor(theme?: DisplayTheme) {
   if (theme === "night") {
     return color("bg-black");
   }
-  if (theme === "transparent") {
```

Comment:
This is a dead code. Explained in https://github.com/metabase/metabase/pull/44275#discussion_r1642409169